types/faker/index.d.ts

```diff
@@ -15,7 +16,6 @@ declare namespace Faker {
 
 		address: {
 			zipCode(format?: string): string;
-			zipCodeByState(state: string): string;
```

Comment:
I see this in the source [here](https://github.com/Marak/faker.js/blob/72dce1198fd348f069b8e42e91ef67b74da1009f/lib/address.js#L39). It's not on the readme but it look like it's in the package. Is this also the case with the other methods? I don't see the 4.1.0 package on npm so I can't confirm.

Reply:
You are comparing to master branch, but for [4.1.0](https://github.com/Marak/faker.js/blob/v4.1.0/lib/address.js) you can check that `zipCodeByState` doesn't exist.